infra/src/main/java/com/redhat/service/smartevents/infra/metrics/MetricsOperation.java

```diff
@@ -0,0 +1,24 @@
+package com.redhat.service.smartevents.infra.metrics;
+
+import io.micrometer.core.instrument.Tag;
+
+/*
+    Constants for the logical user API operations we are report SLI data for.
+ */
+public enum MetricsOperation {
```

Comment:
I moved this to `infra`. It contains _ALL_ operations for which we emit metrics.

It used to be in just `manager` and an equivalent was in `operator`. I've combined and moved.